packages/cms/views/layout.html

```diff
@@ -369,7 +369,7 @@ <h2 class="header-subtitle">
             {% else %}
             <a href="{{data.siteUrl}}/oauth/login"  role="button">
               <!-- <span class="glyphicon glyphicon-user"></span>  !-->
-              {{ __('Mijn account') }}
+              {{ __('Inloggen') }}
```

Comment:
Can we use the text from the desktop login button?

```suggestion
              {% if data.global.myAccountButtonText %} {{ data.global.myAccountButtonText | safe }} {% else %} {{ __('Mijn Account') }} {% endif %}
```